modules/tiled-gallery/tiled-gallery.php

```diff
@@ -16,6 +16,19 @@ public function __construct() {
 		add_action( 'admin_init', array( $this, 'settings_api_init' ) );
 		add_filter( 'jetpack_gallery_types', array( $this, 'jetpack_gallery_types' ), 9 );
 		add_filter( 'jetpack_default_gallery_type', array( $this, 'jetpack_default_gallery_type' ) );
+
+		// Add Tiled Galleries functionality to core's media gallery widget.
+		add_filter( 'widget_media_gallery_instance_schema', array( $this, 'core_media_widget_compat' ) );
+	}
+
+	public function core_media_widget_compat( $schema ) {
+		$schema['type'] = array(
+			'type' => 'string',
+			'enum' => array( 'default', 'rectangular', 'square', 'circle', 'columns', 'slideshow' ),
+			'description' => __( 'Type of gallery.', 'jetpack' ),
```

Comment:
Needs moar `esc_html__`

Reply:
Why? This should be handled when being output depending on the context. If this is being sent back in the REST API as part of JSON, you wouldn't want HTML entities in here.

Reply:
If this is a string that would be displayed in front end sometime, and because of that, `__` was added, then it needs `esc_html__` since it's safer: it would protect against malicious HTML potentially inserted in a translation file.

Reply:
In this case looks like `esc_html` is used when this is rendered as an option's text [here](https://github.com/Automattic/jetpack/blob/764a699120ca6795dac3d264207713516200ee7b/functions.gallery.php#L68).

Reply:
It's safest to practice late-escaping.

Reply:
Thanks oh I see, thanks for pointing that @oskosk 
And yes Weston, I super agree with that, I was out of context since I was missing this piece that Oscar referenced.